### PR TITLE
Censor seesion in the log of ci-secret-bootstrap

### PR DESCRIPTION
**Pull request by hongkailiu**
https://issues.redhat.com/browse/DPTP-880

Local test results:

```bash
$ ci-secret-bootstrap -bw-user aaa@redhat.com --bw-password-path /tmp/_bw_password --config ./core-services/ci-secret-bootstrap/_config.yaml 
INFO[0000] Parsed kubeconfig context: default           
INFO[0000] Parsed kubeconfig context: build01           
INFO[0000] running bw command ...                        args="[login aaa@redhat.com CENSORED --response]"
INFO[0003] running bw command ...                        args="[--session CENSORED list items]"
INFO[0003] running bw command ...                        args="[--session CENSORED get attachment
```

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: cmd/ci-secret-bootstrap/main.go [hunk 1]
```diff
@@ -68,16 +68,26 @@ func (o *options) validateOptions() error {
 	return nil
 }
 
+var (
+	secrets = sets.NewString()
```
Comment (stevekuznetsov) [inline cmd/ci-secret-bootstrap/main.go]: We should not use global variables. If you're threading through a closure to the client anyway, just update a local variable in `main()`.

Comment (hongkailiu) [reply cmd/ci-secret-bootstrap/main.go]: Updated.

**Pull request merged**